src/docs/sdk/data-handling.mdx

```diff
@@ -51,6 +50,43 @@ Some examples of auto instrumentation that could attach sensitive data:
 - Desktop apps including window title.
 - A Web framework routing instrumentation attaching route `to` and `from`.
 
+## Structuring Data
+
+For better data scrubbing on the server side, SDKs should save data in a strucutured way, when possible. Starting point of the discussion was [RFC-0038](https://github.com/getsentry/rfcs/blob/main/text/0038-scrubbing-sensitive-data.md)
+
+### Spans
+
+This helps Relay to know what kind of data it receives and this helps with scrubbing sensitive data.
+
+- `http` spans containing urls:
+
+  The description of spans with `op` set to `http` must follow the format `HTTP_METHOD scheme://host/path` (ex. `GET https://example.com/foo`).
+  If an authority is present in the URL (`https://username:password@example.com`), the authority must be omitted completely.
+  If query strings or fragments are present in the URL, both are set into the data attribute of the span.
+
+  ```js
+  span.setData({
+    "http.query": url.getQuery(),
+    "http.fragment": url.getFragment(),
+  });
+  ```
+
+  Additionally all semantic conventions of OpenTelementry for http spans should be set in the `span.data` if applicable:
+  https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/
+
+- `db` spans containing database queries: (sql, graphql, elasticsearch, mongodb, ...)
```

Comment:
Based on this [issue](https://github.com/getsentry/sentry-python/issues/1784), we might consider to move everything mentioned here below `send_default_pii`, to allow people to opt-out if they have the desire to do so.

Reply:
This will be handled in the RFC concerning how to give users better control of what is happening with personal data, that will be created soon.